segmentstore/server/src/test/java/io/pravega/segmentstore/server/containers/ContainerEventProcessorTests.java

```diff
@@ -87,14 +87,15 @@ public void testBasicContainerEventProcessor() throws Exception {
     public static void testBasicContainerEventProcessor(ContainerEventProcessor containerEventProcessor) throws Exception {
```

Comment:
should we add test to specifically test this change?

Reply:
Yes, I have added one just now.